services/workers/src/main/resources/table-cluster-sqs-spb.xml

```diff
@@ -42,12 +42,12 @@
 					</bean>
 				</property>
 				<property name="targetMethod" value="run" />
-				<property name="concurrent" value="false" />
+				<property name="concurrent" value="true" />
```

Comment:
There is a chance that one machine would take all the semaphore locks defeating the distribution of workers in different machines